### DISCUSSION TITLE
U.S Adult Literacy Rates

### DISCUSSION THREAD

**Discussion by aembar** [jakobzhao/geog495#53] [Final Project]
Application: [https://aembar.github.io/GEOG495AdultLiteracyRateMap/](https://aembar.github.io/GEOG495AdultLiteracyRateMap/)
Repo: [https://github.com/aembar/GEOG495AdultLiteracyRateMap](https://github.com/aembar/GEOG495AdultLiteracyRateMap)
Description: This project visually shows the U.S Adult Literacy Rate per county as an interactive web application. There is a search bar that can be used to find locations in the U.S and the Adult Literacy Rate for that county will be displayed in the upper lefthand corner. If you do not see the data on the map, try refreshing the page. 
<img width="1440" alt="Screen Shot 2021-12-09 at 5 59 48 PM" src="https://user-images.githubusercontent.com/52722485/145513405-de2d0628-b6f0-4aa3-b067-c5b49354b9e2.png">

# Search Bar
In the search bar enter a location to see the Adult Literacy Rate for your county.
<img width="1438" alt="Screen Shot 2021-12-09 at 7 11 37 PM" src="https://user-images.githubusercontent.com/52722485/145513430-76309f54-c086-4c7d-8eb4-5a066bcf6e33.png">

**Comment by LiamGearyUW** [comment]
The map is competently constructed. Choropleth colors are effective and intuitive.

My main issue is that I'm not sure what the map is really telling me. A more detailed description of what "literacy rates" exactly means would be helpful, as the term typically refers to what percentage of the population can read and write. Also, the legend div appears to change widths depending on the width of the window, though the features div doesn't. I recommend making the two uniform in some way.

**Comment by alexanderlyu** [comment]
The colors used for the map are clear and work well, while contrasting with the basemap. Something that could be improved upon is adding borders to the counties in order to better differentiate them.

**Comment by ebar23** [comment]
Your map turned out great in terms of functionality and aesthetic. I do agree with Liam that it would be a good idea to add some sort of minor explanation as to what "literacy rates" mean and how that can be displayed here within the contexts of the map. Overall though, I think you did a great job and I believe that the message that you are trying to communicate as an important one to address. Great job!

**Comment by dldl1901s** [comment]
I really like your choice of base map and color. However, it would have been nice to provide other information such as population density along with the illiteracy rate.

**Comment by VkSath00** [comment]
I like the way your map looks visually, specifically your basemap as it's colorful and pops out to the user. I think it would be useful if you included a small section about what literacy rates truly are and how the calculation to the map works as well. Your map has a nice message though which is great.

**Comment by shimizusUW** [comment]
I love your use of colors and the overall design of the map. Maybe having more context with the map beside the data/numbers can help to improve this application. Yes, I can see the numbers on literacy rates, but there's not much I can do with this. Nor can I tell the relationship between the literacy rate and the county specifically.

**Comment by FaisalTF** [comment]
I really love the aesthetics, besides being pretty to browse through, I think the contrast helps focus on any trends or outliers. Besides the making of this, it's really interesting to see the insights from the data, literacy rate trends and thinking about overall rural vs urban regions. Trying to constructively think about how this could improve, I think the interface could be slightly redesigned for smoother navigation; overlays at three different corners might get slightly distracting - maybe they can be bunched together within a nice UI.
